CMakeLists.txt

```diff
@@ -231,6 +232,7 @@ if(MZ_ZLIB)
     endif()
 
     if(MZ_ZLIB)
+        list(APPEND MINIZIP_DEP_PKG ZLIB)
```

Comment:
Is it possible to use `MINIZIP_DEP` instead? It seems very similar?

Reply:
Should this statement be moved to inside `if(ZLIB_FOUND AND NOT MZ_FORCE_FETCH_LIBS)` ? It seems like we only want to define dependent packages if we are using them. In the case of `MZ_FETCH_LIBS` the library is internally compiled.. 

Same with the rest?

Reply:
> Is it possible to use `MINIZIP_DEP` instead? It seems very similar?

MINIZIP_DEP_PKG is the list of package names for  "find_package/find_dependency" and I think MINIZIP_DEP only gives target names, not the package name.
For e.g.:
ZLIB::ZLIB is the target, not package name, its package name is "ZLIB".  

> Should this statement be moved to inside `if(ZLIB_FOUND AND NOT MZ_FORCE_FETCH_LIBS)` ? It seems like we only want to define dependent packages if we are using them. In the case of `MZ_FETCH_LIBS` the library is internally compiled..

As you might know "find_dependency" checks for the dependency of minizip. So, even if the dependency is internally built we would still require it to find it because its config file is different from minizip.

Reply:
I don't have much experience with `find_dependency`.

Reply:
it is just as a check for dependencies before including package